### PR TITLE
blog: release announcement for Arquillian cube 1.15.3

### PR DESCRIPTION
**Pull request by dipak-pawar**

### CODE REVIEW

**Comment by dipak-pawar** [comment]
screenshot from 2018-04-04 19-59-57

File: blog/arquillian-cube-1.15.3.textile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Here's an example:
+
+bc(prettify).. <extension qualifier="openshift">
+    <property name="cube.fmp.build">true</property>
+</extension>
+
+p. By enabling this property, cube will build images, generate resources using @fabric8-maven-plugin@ when running test from @Maven@ or @IDE@.
+
+p(info). %User can configure to run @fabric8-maven-plugin@ only from @IDE@ (not from maven build) by setting property @cube.fmp.build.disable.for.mvn@ to @true@. Look "here":http://arquillian.org/arquillian-cube/#_fabric8_maven_plugin_integration_with_cube for more configuration options.%
```
Comment (lordofthejars) [inline blog/arquillian-cube-1.15.3.textile]: Is property `true` by default? Maybe it should be true?

Comment (dipak-pawar) [reply blog/arquillian-cube-1.15.3.textile]: no there is no point to make it true. we need to embedded maven build only if user is using fmp.

Comment (lordofthejars) [reply blog/arquillian-cube-1.15.3.textile]: So by default it runs from IDE and maven build. Ok.

Comment (dipak-pawar) [reply blog/arquillian-cube-1.15.3.textile]: notice that there might be a situation when a user doesn't have fmp in his pom.xml. In this case, we should not execute embedded maven build from cube.

**Comment by dipak-pawar** [comment]
@lordofthejars it's already there https://github.com/arquillian/arquillian.github.io/pull/430/files#diff-93f517b51ea2b7af3a435e302f6b0552R37

**Comment by lordofthejars** [comment]
I mean the `fmp` integration

**Pull request merged**